lms/djangoapps/verify_student/management/commands/trigger_softwaresecurephotoverifications_post_save_signal.py

```diff
@@ -0,0 +1,67 @@
+"""
+Django admin command to save SoftwareSecurePhotoVerifications given an iterable of
+verification_ids, thereby re-emitting the post_save signal.
+"""
+
+import logging
+import time
+
+from django.core.management.base import BaseCommand
+
+from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Command to save SoftwareSecurePhotoVerification model instances based on a provided interable of verifiction_ids.
+    This re-emits the post_save signal.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--verification-ids',
+            dest='verification_ids',
+            action='store',
+            nargs='+',
+            type=str,
+            help='SoftwareSecurePhotoVerification IDs to save'
+        )
+        parser.add_argument(
+            '--batch_size',
+            action='store',
+            dest='batch_size',
+            type=int,
+            default=300,
+            help='Maximum number of SoftwareSecurePhotoVerifications to process. '
+                 'This helps avoid overloading the database while updating large amount of data.'
+        )
+        parser.add_argument(
+            '--sleep_time',
+            action='store',
+            dest='sleep_time',
+            type=int,
+            default=10,
+            help='Sleep time in seconds between update of batches'
+        )
+
+    def handle(self, *args, **options):
+        verification_ids = options['verification_ids']
+        batch_size = options['batch_size']
+        sleep_time = options['sleep_time']
+
+        count = 0
+        verifications = SoftwareSecurePhotoVerification.objects.filter(id__in=verification_ids)
```

Comment:
To make this easier, why not just retrieve all of the approved or denied records since Oct 8th and retrigger the save action?
That way, you don't need to rely on passed in value.